stream-chat-android-client/src/main/java/io/getstream/chat/android/client/extensions/StringExtensions.kt

```diff
@@ -16,7 +16,15 @@
 
 package io.getstream.chat.android.client.extensions
 
+import android.net.Uri
```

Comment:
New methods on this flie are simple "input/output method", could we cover them with UnitTest?

Reply:
Added tests in the following commit: https://github.com/GetStream/stream-chat-android/pull/4600/commits/873f11fe673e0694b7551cfc59c23fffbcb13432